src/2ls/summary_checker_base.h

```diff
@@ -57,10 +56,6 @@ class summary_checker_baset:public property_checkert
     ssa_inliner.set_message_handler(_message_handler);
   }
 
-  // statistics
-  absolute_timet start_time;
```

Comment:
Has this been unused?
If not, use `#include <chrono>` instead.

Reply:
Yes, this wasn't used anywhere (perhaps it had been used some time ago somewhere but it may have been removed).